src/seeds.onboarding.cpp

```diff
@@ -389,11 +395,62 @@ void onboarding::accept(name account, checksum256 invite_secret, string publicKe
   accept_invite(account, invite_secret, publicKey, string(""));
 }
 
+void onboarding::chkcleanup() {
+  require_auth(get_self());
+
+  timestamp_tables timestamps(get_self(), get_self().value);
+  invite_tables invites(get_self(), get_self().value);
+
+  if (invites.begin() == invites.end()) { return; }
+
+  if (timestamps.begin() != timestamps.end()) {
+
+    auto titr = timestamps.end();
+    titr--;
+
+    uint64_t now = eosio::current_time_point().sec_since_epoch();
+
+    // uncomment this line for tests
+    // if (titr->timestamp + 2 > now) { return; }
+    if (titr->timestamp + (utils::seconds_per_day * 14) > now) { return; }
+
+    uint64_t start_id = 0;
+    uint64_t max_id = titr->invite_id - 1;
+
+    if (titr->id > 0) {
+      auto prev_itr = titr;
+      prev_itr--;
+
+      if (prev_itr->invite_id < max_id) {
+        start_id = prev_itr->invite_id;
+      }
+    }
+
+    action(
+      permission_level(get_self(), "active"_n),
+      get_self(),
+      "cleanup"_n,
+      std::make_tuple(start_id, max_id, config_get("batchsize"_n))
```

Comment:
not sure cleanup can handle 200...

Reply:
actually should be fine, it's x8 for each returned invite... 25 returned invites hopefully works